### ISSUE TITLE
[PickerInput]: [x] button in search field is not available using keyboard

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1552]
### Description
"x" remove buttons in multi select combobox with selected options are not available using keyboard

### Precondition
Screen Reader is **OFF** or **ON**

### Steps to Reproduce
1. Open [Picker Input](https://uui.epam.com/documents?id=pickerInput&mode=doc&skin=UUI4_promo&category=components) component.
2. Use the `Tab` key to navigate to the Multi select combobox and select any options.
3. Use the `Tab` key to navigate to each "x" button of each selected option in combobox.

### Actual Result
"x" remove buttons in combobox with selected options are not available using keyboard.

https://github.com/epam/UUI/assets/57751283/f0b50a84-6c94-4cfe-9dc4-bb13290efba0

### Expected Result
Each "x" remove button in combobox with selected options shrould be available using keyboard, should have visible focus indicator and should have descriptive accessible name, e.g., "Remove A1", "Remove A2+" etc., "Clear All" for "x" button that clear all results.
![Unknown](https://github.com/epam/UUI/assets/57751283/4dce99f1-306f-48f1-b948-0683e6d8c1f0)

**Failed WCAG Checkpoint:** 2.1.1 Keyboard (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by NatalliaAlieva** [comment]
After some research, we came to the conclusion that there is no need to implement the ability to delete a value from the keyboard by clicking on the [x] icon.

[Here](https://paste.twilio.design/components/multiselect-combobox) is a similar example of Multiselect Combobox implementation.

User can delete values by opening back to the Picker Input and unselecting the previously selected value.

https://github.com/epam/UUI/assets/57751283/6cbb0257-e960-4111-81a4-454f492936d7

**Comment by NatalliaAlieva** [comment]
Possible implementation option: allow User to select values with the `left`/`right` arrows and delete value with the `Delete` key. Such implementation is presented in these cases:
1) https://paste.twilio.design/components/multiselect-combobox
2) https://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/disabled-items/
